src/cdn_definitions/data.yaml

```diff
@@ -189,3 +189,8 @@ signing_keys_mappings:
 # A list of RHEL releasevers (minor versions) that should be excluded from listing files
 exclude_from_listings:
 - "0.0"
+
+# Major RHEL versions which are still expecting new minor releases in
+# future. Versions not listed here are exempt from various tooling.
+rhel_open_dist:
```

Comment:
At the top of this file there is a:

```
version: "1.1.0"
```

Can you please bump that to:

```
version: "1.2.0"
```

This is the version for the data format and it follows semver rules, so introducing a new field and not making any backwards-incompat changes means bumping the middle number.

Reply:
thank you, version bumped